doc/releases/migration-guide-3.7.rst

```diff
@@ -92,6 +92,17 @@ MbedTLS
   of other Kconfig options which are now named `CONFIG_MBEDTLS_SHA*`.
 * The `CONFIG_MBEDTLS_MAC_ALL_ENABLED` Kconfig option has been removed. Its equivalent is the
   combination of :kconfig:option:`CONFIG_MBEDTLS_HASH_ALL_ENABLED` and :kconfig:option:`CONFIG_MBEDTLS_CMAC`.
+* Choice symbols :kconfig:option:`CONFIG_MBEDTLS_PSA_CRYPTO_LEGACY_RNG` and
+  :kconfig:option:`CONFIG_MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG` were added to allow the user
+  to specify how Mbed TLS PSA crypto core should generate random numbers. The former
+  option, which is the default, relies on legacy entropy and ctr/hmac DRBGs modules,
+  while the latter relies on CSPRNG drivers. (:github:`71827`)
```

Comment:
Wondering, is this entry relevant to the migration guide, or only to the release notes?